### ISSUE TITLE
Console Error Relating to Preview Images

### ISSUE THREAD

**Issue by Nicholas-Westby** [kgiszewski/Archetype#414]
Noticed this when I was in the content section and viewed a content node that had some Archetypes on it:

![preview-image-not-found](https://user-images.githubusercontent.com/5933222/27806979-b8f60826-5ff2-11e7-9e27-dd08b70dfdad.png)

This was upgraded from Archetype 1.13.2 to 1.14.1. For the search engines, the error text is:

```
GET http://localhost:54314/umbraco/%7B%7BoverlayMenu.fieldsetPreview.previewImage%7D%7D 404 (Not Found)
```

Seems like it's related to the new fieldset preview functionality, and that this attempting to evaluate an Angular expression as a resource (e.g., an image, script, or style).

**Comment by kjac** [comment]
@Nicholas-Westby thank for reporting. We'll get this fixed for the next release. No release date as of yet, kinda waiting for additional issues to fix :)

**Comment by kjac** [comment]
@Nicholas-Westby I can't seem to reproduce this :( did you specify some preview images or is it just happening for no good reason?

**Comment by kjac** [comment]
@Nicholas-Westby disregard the last comment - Chrome was helping me by filtering the error out.

**Comment by kjac** [comment]
Fixed. Reopening the issue 'till we have a new release ready.

**Comment by kjac** [comment]
Fixed in v1.15.1
